client/packages/android/README.MD

```diff
@@ -108,14 +118,42 @@ yarn android:build:debug
 yarn android:build:release
 ```
 
-To make the an apk that you can install on a device for testing, run `yarn android:build:debug` in the client directory. You will find the resulting apk in `client/packages/android/app/build/outputs/apk/debug`, and you can then transfer it to the tablet and install it. This can be done by uploading the apk file via the android studio device manager, then clicking the apk file from the device itself. Or using [adb](https://developer.android.com/tools/adb)
-
 ### Known script issues
 
 When building apk `properties.local` files is required (it can be empty). Also can sometime get a lock on gradle build, can run `find ~/.gradle -type f -name "*.lock" -delete` to fix it
 
+If encountering errors such as "Failed to find tool." Check the `server/android/.cargo/config.toml` file has the correct targets.
+
+## Building APKs
+
+There are two types of apks that can be built locally - Debug and Release. See below sections for each type
```

Comment:
Can you mention here somewhere, about the gradle configuration (it's something that we never configured before, so would be great to add)